src/js/port_handler.js

```diff
@@ -23,6 +23,9 @@ PortHandler.initialize = function () {
     this.selectList = document.querySelector(portPickerElementSelector);
     this.initialWidth = this.selectList.offsetWidth + 12;
 
+    ConfigStorage.get('showVirtualMode', res => self.showVirtualMode = res.showVirtualMode);
```

Comment:
Seems that now we cant show virtual mode connection available.

Reply:
You have to enable Virtual Mode in options tab and restart configurator.

Reply:
I tried that with no succes @haslinghuis

Reply:
Thanks for testing. Reverted the change in Porthandler and now it works again. I'd moved the two lines out of the check because it's being called twice a second to reduce workload.